docs/release-notes/_release-notes-0-17-15.md

```diff
@@ -19,14 +19,38 @@ The [x](../account-management/plans/) section was created as a result of this ch
 
 The following features updates are included in this release.
 
-### SQL Support Updates - Beta
+### Macrometa Stores API Keys
+
+In previous versions, when a user created an API key using the `POST /_api/key` API, Macromata returned the newly-created API key in the response on successful execution. Internally, the API key was not stored, and there was no way to get the API key again.
+
+As of this release, Macrometa now stores new API keys created using the `POST /_api/key API`. Due to this change, Macrometa now returns the API key in response to `GET /_api/key/{keyid}`.
```

Comment:
I'm unsure whether customers should know that we store the API Keys. I think it'll be good to just mention that they can now retrieve the newly generated API Keys using GET REST API, which was not possible earlier. IMO, lets not reveal whether API Keys are stored internally or generated using some calculations or some other way. 
Reason - storing API Key might be considered as security risk.
I'm fine with current doc change too if security experts are ok with it.

Reply:
@eddie-macrometa and @elof, can you provide some guidance here?

Reply:
I suggest: As of this release, an API key created using `POST /_api/key` can now be retrieved using `GET /_api/key/{keyid}`

Reply:
Content updated, please review.